Utilities/Templates/Extensions/Default/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(EXTENSION_CONTRIBUTORS "John Doe (AnyWare Corp.)")
 set(EXTENSION_DESCRIPTION "This is an example of a simple extension")
 set(EXTENSION_ICONURL "http://www.example.com/Slicer/Extensions/TemplateKey.png")
 set(EXTENSION_SCREENSHOTURLS "http://www.example.com/Slicer/Extensions/TemplateKey/Screenshots/1.png")
-set(EXTENSION_DEPENDS "NA") # Specified as a space separated string, a list or 'NA' if any
+set(EXTENSION_DEPENDS "NA") # Specified as a list or 'NA' if any
```

Comment:
This would be more clear:

... or "NA" if the extension does not depend on other extensions

... or "NA" if no dependencies